Doc/library/csv.rst

```diff
@@ -318,18 +332,22 @@ The :mod:`csv` module defines the following exception:
 
 .. _csv-fmt-params:
 
-Dialects and Formatting Parameters
+Custom Dialects
 ----------------------------------
 
-To make it easier to specify the format of input and output records, specific
-formatting parameters are grouped together into dialects.  A dialect is a
-subclass of the :class:`Dialect` class having a set of specific methods and a
```

Comment:
Looking at this now, the original docs look better than my former proposed changes in this spot... I will revise this